help/C/gnucash-help.xml

```diff
@@ -343,7 +343,7 @@
       </author>,
       <author>
         <firstname>David</firstname> <surname>Cousens</surname>
-      </author>,
+      </author>
```

Comment:
Why do you want to remove the comma? It is a long comma separated list of authors: https://code.gnucash.org/docs/C/gnucash-help/authors.html